ibis/__init__.py

```diff
@@ -37,7 +37,7 @@
 
 with suppress(ImportError):
```

Comment:
Can you try to temporary remove this with? It may be hiding the actual error.

Reply:
Done. New commit pushed to the PR.